### ISSUE TITLE
Unsupported style property padding-left.

### ISSUE THREAD

**Issue by HriBB** [kadirahq/storybook-ui#46]
I am getting this error with the latest `storybook`.

```
Warning: Unsupported style property padding-left. Did you mean paddingLeft? Check the render method of `Stories`.
```

Obviously [this](https://github.com/kadirahq/storybook-ui/blob/master/src/modules/ui/components/left_panel/stories.js#L10) needs to be `paddingLeft` not `padding-left`.

**Comment by arunoda** [comment]
Oh yes.
@mnmtanish

**Comment by PEM--** [comment]
Fixed with #52
